cmd/agent/api/pb/api.proto

```diff
@@ -15,12 +15,18 @@ service Agent {
 }
 
 service AgentSecure {
-    rpc GetTags (TagRequest) returns (TagReply) {
+    rpc StreamTags(StreamTagsRequest) returns (stream StreamTagsResponse) {
```

Comment:
I'm wondering if we should use more similar name for the 2 service linked the tagger.

```golang 
rpc TaggerStreamEntities(TaggerStreamEntitiesRequest) returns (stream TaggerStreamEntityResponse) {...}

rpc TaggerFetchEntity(TaggerFetchEntityRequest)  return (TaggerEntityResponse) {...}
```

WDYT?